reports/technical_report/5_conclusions.md

```diff
@@ -60,16 +60,18 @@ Finally, we would like to incorporate additional secondary data into our analysi
 ### Micro analysis
 
 There are two principle significant investments required to improve the data quality of Covid-19 notices.
-Firstly, the data collected by Glass must be more reliable and complete: notices that are truncated must not be truncated; notices containing snippets from different sections of a businesses website must be separated; the time and date of collection must be provided rather than just a month.
+Firstly, the data collected by Glass must be more reliable and complete: notices that are truncated must not be truncated; notices containing snippets from different sections of a businesses website must be separated; and the time and date of collection must be provided rather than just a month.
 Unfortunately improved data collection would have to start again and would not be able to recover historic information, thus losing a key feature of the data.
 Secondly, better processing of the notices must be performed such that irrelevant notices are filtered out and relevant notices are classified into several categories such as "Business as usual", "Temporary closure", "Permanent closure", "Adaptation", "Partial closure" etc.
-This would likely require hand-annotation of the relevant sections of a subset of notices; training a model on these labels; extrapolating to the rest of the notices using the model; and performing a validation study on the results.
+This would require hand-annotation of a subset of notices; training a model on these labels; extrapolating to the rest of the notices using the model; and performing a validation study on the results.
 [Recent analysis](https://datasciencecampus.ons.gov.uk/extracting-text-data-from-business-website-covid-19-notices/)
```

Comment:
Maybe also add a reference? Also note that the hyperlinks in the text are currently not coloured so hard to distinguish from plain text.